README.md

```diff
@@ -71,6 +71,8 @@ The `CommonFactory` reads a Cradle configuration from the cradle.json file.
 * cradleInstanceName - this option defines a special identifier that divides data within one keyspace with infra as the default value.
 * cradleMaxEventBatchSize - this option defines the maximum event batch size in bytes with its default value set to 1048576.
 * cradleMaxMessageBatchSize - this option defines the maximum message batch size in bytes with its default value set to 1048576.
+* timeout - this option defines connection timeout in millisecons. If set to 0 or ommited, the default value 5000 is used.
```

Comment:
timeout - this option defines connection timeout in **milliseconds**. If set to 0 or ommited, the default value **of** 5000 is used.